src/nylas-connection.js

```diff
@@ -22,8 +22,9 @@ const SDK_VERSION = PACKAGE_JSON.version;
 const SUPPORTED_API_VERSION = '2.0';
 
 module.exports = class NylasConnection {
-  constructor(accessToken) {
+  constructor(accessToken, { clientId }) {
```

Comment:
Could we just pass in the client ID as a standalone parameter? What's the purpose of the destructuring?

Reply:
I would prefer if the constructor had only one argument: an object with keys for the different parameters. That's the best way to handle named function arguments in Node, because it's future-proof: your code can only reference the keys that it's interested in.

Unfortunately, we can't change the first argument to an object without breaking backwards compatibility, but we can make sure that any other arguments that need to be added are keys in the object. Does that make sense?